features/parameter-types/rooms.js

```diff
@@ -33,8 +33,8 @@ class Room {
 // See: https://github.com/zinc-collective/convene/issues/41
 defineParameterType({
   name: "accessLevel",
-  regexp: /(an |a )?(Unlocked|Internal|Locked)/,
-  transformer: (level) => new AccessLevel(level),
+  regexp: /(an |a |the )?(Unlocked|Internal|Locked)/,
```

Comment:
Not blocking: We may want to remove the `?` from this regex, so that we don't have to deal with different numbers of arguments being passed into the Transformer.